flux_exchange.F90

```diff
@@ -1365,6 +1365,16 @@ subroutine flux_exchange_init ( Time, Atm, Land, Ice, Ocean, Ocean_state,&
     allocate( ice_ocean_boundary%calving_hflx  (is:ie,js:je) ) ;    ice_ocean_boundary%calving_hflx = 0.0
     allocate( ice_ocean_boundary%p        (is:ie,js:je) ) ;         ice_ocean_boundary%p = 0.0
     allocate( ice_ocean_boundary%mi       (is:ie,js:je) ) ;         ice_ocean_boundary%mi = 0.0
+    
+    if (Ice%Ice_state%do_icebergs) then
+      !These lines might not compile with SIS (rather than SIS2)
+      if (associated(Ice%ustar_berg)) &
+        allocate( ice_ocean_boundary%ustar_berg  (is:ie,js:je) ) ;      ice_ocean_boundary%ustar_berg = 0.0
```

Comment:
Note that these would require modifying all the existing ocean models (mom5, GOLD, slab as well as MOM6) to add the extra ice_ocean_boundary (dummy) members %ustar_berg,%area_berg,%mass_berg .